ui/raidboss/data/05-shb/raid/e9n.js

```diff
@@ -225,20 +225,90 @@ export default {
       'missingTranslations': true,
       'replaceSync': {
         'Cloud Of Darkness': 'Wolke der Dunkelheit',
+        'Stygian Arbor': 'stygisch[a] Dorn',
```

Comment:
XIVAPI tell me this, but it looks weird.

Reply:
When I enter the dungeon with DE, didn't see the `[a]` letter.
Don't know why but delete this would match the NPC.